### PR TITLE
seed key-server stats

### PR DESCRIPTION
**Pull request by whaught**
Issue https://github.com/google/exposure-notifications-verification-server/issues/1512

## Proposed Changes

* Generate random data to seed key-server stats for development

```release-note
Add random key-server stats data in dev seed script
```

### CODE REVIEW

File: tools/seed/main.go [hunk 1]
```diff
@@ -413,6 +417,43 @@ func generateCodesAndStats(db *database.Database, realm *database.Realm) error {
...
+// generateKeyServerStats generates stats normally gathered from a key-server. This is
+// primarily used to test statistics and graphs.
+func generateKeyServerStats(db *database.Database, realm *database.Realm) error {
+	if err := db.SaveKeyServerStats(&database.KeyServerStats{RealmID: realm.ID}); err != nil {
+		return fmt.Errorf("failed create stats config: %w", err)
+	}
+
+	nowish := time.Now().Add(-5 * time.Minute)
+	for day := 0; day < 30; day++ {
+		date := nowish.Add(time.Duration(day) * -24 * time.Hour)
```
Comment (mikehelmick) [inline tools/seed/main.go:429]: I think we should UTC midnight this (or the one on 427)

Comment (whaught) [reply tools/seed/main.go:429]: UTCed - not sure if midnight really helps

Comment (mikehelmick) [reply tools/seed/main.go:429]: in production, they'll be UTC midnight for days

Comment (mikehelmick) [reply tools/seed/main.go:429]: that's what you'll get from the key server

**Review by sethvargo — APPROVED** [review]

File: tools/seed/main.go [hunk 2]
```diff
@@ -413,6 +417,45 @@ func generateCodesAndStats(db *database.Database, realm *database.Realm) error {
...
 
+// generateKeyServerStats generates stats normally gathered from a key-server. This is
+// primarily used to test statistics and graphs.
+func generateKeyServerStats(db *database.Database, realm *database.Realm) error {
+	if err := db.SaveKeyServerStats(&database.KeyServerStats{RealmID: realm.ID}); err != nil {
+		return fmt.Errorf("failed create stats config: %w", err)
+	}
+
+	year, month, day := time.Now().UTC().Date()
+	midnight := time.Date(year, month, day, 0, 0, 0, 0, time.UTC)
```
Comment (mikehelmick) [inline tools/seed/main.go:428]: timeutils.UTCMidnight(time.Now())

Comment (whaught) [reply tools/seed/main.go]: ...makes sense that there's a util for this

**Review by mikehelmick — APPROVED** [review]

**Pull request merged**